src/Components/Reports/Common/SystemsFirstPage.js

```diff
@@ -1,22 +0,0 @@
-import React, { Fragment } from 'react';
-import propTypes from 'prop-types';
-import { Section, Column } from '@redhat-cloud-services/frontend-components-pdf-generator';
```

Comment:
As we are no longer using these packages -`@react-pdf/renderer` and 
`@redhat-cloud-services/frontend-components-pdf-generator`, we can remove these from the package.json file.

Reply:
nice catch @r14chandra, removed and pushed the changes. 💯